desktop/components/has_seen/dismisser.coffee

```diff
@@ -1,20 +1,24 @@
 _ = require 'underscore'
 Cookies = require '../cookies/index.coffee'
 
+# Get around inabilty to stub `document`
+domain = document.domain if typeof document isnt 'undefined'
+
 module.exports = class Dismisser
   defaults:
     limit: 5
     expires: 31536000 # 1 Year
+    domain: domain
```

Comment:
Right, but I think that our browser/library is already doing what we want. Running `Cookies.set('sarah-test', 'abc')` (i.e., no domain specified) while visiting artsy.net appears to set a cookie with the correctly scoped domain:

![image](https://cloud.githubusercontent.com/assets/2081340/23678005/51a6e108-0350-11e7-90a9-e61213fefeee.png)